CHANGES.txt

```diff
@@ -61,6 +61,11 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
       expected bytecodes in ActionTests.py.
 
 
+  From Andrew Morrow:
```

Comment:
Move this up a bit, should be alphabetical by last name.

Reply:
Done